### DISCUSSION TITLE
Peak Set idx column

### DISCUSSION THREAD

**Discussion by ahyeonhwang** [GreenleafLab/ArchR#1307] [Questions / Documentation] [answered]
Hello,

I am a little confused on what the idx column in the peak set granges object signifies. I have 443973 total peaks but only 39424 unique idx values. Is idxATAC (1 to 443973) the index for the peaks in the total peak set?

Thank you very much for your help!

Best,
Ahyeon

**Comment by Zepeng-Mu** [comment, marked as answer]
I actually had the same question before: https://github.com/GreenleafLab/ArchR/issues/1209
Turned out the `idx` is only unique within each chromosome, and this is clear if you look the code.
I'm not sure about `idxATAC` for some reason I don't have that in my peakSet.

**Reply by ahyeonhwang** [reply]
Thank you!

**Comment by rcorces** [comment]
related https://github.com/GreenleafLab/ArchR/issues/368

idxATAC shows up in other analyses like peak2gene linkages.

**Reply by ahyeonhwang** [reply]
Thank you!
